submissions/dmytro-botnarenko/html-css-popup/style.css

```diff
@@ -0,0 +1,154 @@
+*,
+*:before,
+*:after {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  box-sizing: border-box;
+}
+html,
+body {
+  height: 100%;
+  background-color: white;
+  font-family: Tahoma, Helvetica, Arial, sans-serif;
+  color: #fff;
+  font-size: calc(12px + 0.39vw);
+  line-height: 24px;
+  letter-spacing: 0.05em;
+}
+a {
+  color: #794f45;
+  text-decoration: none;
+}
+.header {
+  padding: 12px 14px;
+  box-shadow: 0px 1px 2px 1px rgba(0, 0, 0, 0.2);
+}
+.navigation__menu {
+  display: flex;
+  justify-content: end;
+}
+.navigation__item {
+  min-height: 45px;
+  display: flex;
+  flex: 0 1 85px;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  margin-left: 10px;
+}
+.navigation__link {
+  display: flex;
+  justify-content: center;
+  padding: 2px 5px;
+  min-height: 30px;
+  min-width: 95%;
+}
+.navigation__link:hover {
+  border-radius: 3px;
+  box-shadow: 0 0 2px 2px rgba(0, 0, 0, 0.2);
+}
+.navigation__image {
+  width: 27%;
```

Comment:
this approach is not working well for small screens. icons are too small
![image](https://user-images.githubusercontent.com/21179601/187511978-887ba5b0-3644-4230-a51b-99e021fc8391.png)